mitodl/mit-open
# about page updates

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/4641

### Description (What does it do?)
This PR adds some updates the copy and styles on the About Us page. The initial version of the About Us page had the wrong typography rules based on a miscommunication in Figma. The sub headings have been updated to `h4` style, and the body text is now `body1`. Previously in Figma the headings were `h5` and the body text was `body3`. I also updated these styles on the terms of service and privacy policy pages.

Document fragment identifiers were also added to the headers of the various About Us page sections, and a link was added to the home page search box on the "Non-Degree Education" text that links to the appropriate section of the About Us page.

### Screenshots (if appropriate):
![image](https://github.com/mitodl/mit-open/assets/12089658/5ec02a6d-ad14-4b6f-9d76-cdbbbd7c733b)
![image](https://github.com/mitodl/mit-open/assets/12089658/b75ef935-89c9-4c75-ba17-3975cf401183)

### How can this be tested?
 - Spin up `mit-open` on this branch
 - Visit the home page at http://localhost:8063/
 - Click the "Non-Degree Education" link
 - Verify that you are brought to http://localhost:8063/about#non-degree-learning
 - Verify that the design of the About Us page looks as it should according to the design linked in the issue about at both desktop and mobile resolutions
